.github/workflows/try-runtime.yml

```diff
@@ -0,0 +1,40 @@
+name: Run try-runtime on storage migration label
+
+on:
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  run_if_label_matches:
+    if: github.event.label.name == 'migration'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+
+      - name: Setup worker
+        uses: "./.github/templates/setup-worker"
+
+      - name: Cache Build artefacts
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
+          key: ${{ runner.os }}-cargo-release-${{ env.POLKA_VERSION }}-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: ${{ runner.os }}-cargo-release-${{ env.POLKA_VERSION }}
+
+      - run: 
+          echo "Found label runtime_migration. Running tests"
+          export RUST_LOG=remote-ext=debug,runtime=debug
+          echo "---------- Running try-runtime for Trappist ----------"
+          time cargo build --locked --release -p trappist-runtime
```

Comment:
I don't see why you need to build this runtime explicitly since the very next line will do it anyway. If you want to compile only trappist runtime, you can set it up [this](https://github.com/paritytech/trappist/blob/6168897c8811e7ea2f637b2f041e7313cb05f8e2/.github/workflows/check.yml#L80) way

Reply:
For this purpose we do not need the `stout-runtime`.

I tried:
```
cargo build --workspace --exclude stout-runtime  --locked --release
```
And for some reason it still builds the `stout-runtime`:
```
   Compiling rococo-runtime v0.9.37 (https://github.com/paritytech/polkadot?branch=release-v0.9.37#64572398)
   Compiling polkadot-runtime v0.9.37 (https://github.com/paritytech/polkadot?branch=release-v0.9.37#64572398)
   Compiling kusama-runtime v0.9.37 (https://github.com/paritytech/polkadot?branch=release-v0.9.37#64572398)
   Compiling trappist-runtime v1.0.0 (/home/bruno/src/trappist/runtime/trappist)
   Compiling stout-runtime v1.0.0 (/home/bruno/src/trappist/runtime/stout)
  Building [====================>  ] 1041/1106: stout-runtime(build), rococo-runtime(build), polkadot-runtime(build), trapp...
```
Any thoughts?

Reply:
I believe you need to compile the runtime explicitly because on the next line when we run `cargo run ...` we have to pass in the runtime wasm:
```
--runtime ./target/release/wbuild/trappist-runtime/target/wasm32-unknown-unknown/release/trappist_runtime.wasm
```

Reply:
You are right. I am not sure why Polkadot has it this way:
- https://github.com/paritytech/polkadot/blob/6a71a7ef7dbe4968db09d94305ae28d9bd95782d/scripts/ci/gitlab/pipeline/check.yml#L71

If we are calling `cargo run..` on the very next line that should be sufficient.

Reply:
Yea i dont think we need the first line. Can you remove it and try?

Reply:
hmm looks like it worked

Reply:
> If we are calling `cargo run..` on the very next line that should be sufficient.

Like I said in the first comment.

> I tried:
>
> cargo build --workspace --exclude stout-runtime  --locked --release
>
> And for some reason it still builds the stout-runtime:

You only excluded the stout runtime from the workspace. You need to set feature flags properly.

Reply:
try this: 
`cargo run -p trappist --locked --release --no-default-features --features trappist/trappist-runtime,try-runtime try-runtime --runtime ...`